internal/pipe/brew/template.go

```diff
@@ -36,15 +38,31 @@ class {{ .Name }} < Formula
   if OS.mac?
     {{- if .MacOS.DownloadURL }}
     url "{{ .MacOS.DownloadURL }}"
-	{{- if .DownloadStrategy }}, :using => {{ .DownloadStrategy }}{{- end }}
+    {{- if .DownloadStrategy }}, :using => {{ .DownloadStrategy }}{{- end }}
     sha256 "{{ .MacOS.SHA256 }}"
     {{- end }}
   elsif OS.linux?
-    {{- if .Linux.DownloadURL }}
-    url "{{ .Linux.DownloadURL }}"
-	{{- if .DownloadStrategy }}, :using => {{ .DownloadStrategy }}{{- end }}
-    sha256 "{{ .Linux.SHA256 }}"
-    {{- end }}
+    if Hardware::CPU.arm?
```

Comment:
maybe add an `if or .Arm.DownloadURL .Arm64.DownloadURL` to prevent printing this if when no arm/arm64 build exist?

Reply:
Done